test/bot-events.test.js

```diff
@@ -141,4 +141,47 @@ describe('Bot Events', function() {
     secondCall.args[0].should.equal('Channel not found in config, not joining:');
     secondCall.args[1].should.equal(channel);
   });
+
+  it('should send joins to slack', function() {
+    var a = this.bot.nickname;
```

Comment:
The initial commits had more readable naming convention, but the unreasonable
restrictions of line length and white space requirements on your automated build
prevent proper naming convention.

On Tue, Oct 06, 2015 at 11:07 AM, Martin Ek < notifications@github.com [notifications@github.com] > wrote:
In test/bot-events.test.js
[https://github.com/ekmartin/slack-irc/pull/59#discussion_r41300198] :

> @@ -141,4 +141,47 @@ describe('Bot Events', function() {
>      secondCall.args[0].should.equal('Channel not found in config, not joining:');
>      secondCall.args[1].should.equal(channel);
>    });
> +
> +  it('should send joins to slack', function() {
> +    var a = this.bot.nickname;


The tests would be a lot more readable if you use more descriptive variable
names instead of a , b and c .

—
Reply to this email directly or view it on GitHub
[https://github.com/ekmartin/slack-irc/pull/59/files#r41300198] .[https://github.com/notifications/beacon/AF6-kCT7eWBKhvi0QN_3T0yK_BdsYotmks5o5AVhgaJpZM4GIfJ5.gif]